fuel/app/classes/presenter/welcome/hello.php

```diff
@@ -3,19 +3,18 @@
 /**
  * The welcome hello presenter.
  *
- * @package  app
  * @extends  Presenter
  */
-class Presenter_Welcome_Hello extends Presenter
+class presenter_welcome_hello extends Presenter
```

Comment:
__[code_sniffer]__
* Class name must begin with a capital letter
* Class name is not valid; consider Presenter_Welcome_Hello instead